anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -149,6 +149,25 @@ let handleExport
 
   File.WriteAllText(outputPath, output)
 
+let handleFindNulls
+  {
+    FindNullsRequest.InputPath = inputPath
+    FindNullsRequest.AidColumn = aidColumn
+  }
+  =
+  let anonParams = AnonymizationParams.Default
+
+  let query =
+    $"""
+      SELECT %s{aidColumn}
+      FROM table
+      WHERE %s{aidColumn} is NULL or %s{aidColumn} = ''
```

Comment:
Or `handleAidCheck` if you prefer it more.

Reply:
hm, are you sure though?

That feels like coupling the AID check with the particular data source underneath. If we support anything else than CSV, this will need to change.

Not that I want to future-proof things, but I'd prefer to keep this check generic and independent of the data source. WDYT?

(regardless of the above, agreed on the `handleAidCheck` name change, since we're more generic than just nulls :+1: )

Reply:
I don't expect we will add support for additional data sources anytime soon (or ever, likely).
But I don't feel strongly about it, leaving it as is is fine with me.